README.md

```diff
@@ -152,26 +130,55 @@ const a = {
   count: ref(0),
 }
 const b = reactive({
-  list: [a],
+  list: [a], // `a.count` will not unwrap!!
 })
 
+// no unwrap for `count`, `.value` is required
 b.list[0].count.value === 0 // true
 ```
 
 ```js
 const b = reactive({
   list: [
     {
-      count: ref(0),
+      count: ref(0), // no unwrap!!
     },
   ],
 })
 
+// no unwrap for `count`, `.value` is required
 b.list[0].count.value === 0 // true
 ```
 
 </details>
 
+<details>
+<summary>
+✅ <b>Should</b> always use <code>ref</code> in a <code>reactive</code> when working with <code>Array</code>
```

Comment:
Ah, do you mean "If you need to use ref in Array, wrap it with `reactive`"?

Reply:
This is the message we had before :D 
https://github.com/vuejs/composition-api/tree/556292d0d60220722fac2cd401b1ba061d6b05ba#ref-unwrap

Reply:
Not a native English speaker, which statement makes more sense to you? We can improve other statements along the way.